rpi-update

```diff
@@ -296,13 +296,27 @@ function noobs_fix {
 
 }
 
+echo " *** Raspberry Pi firmware updater by Hexxeh, enhanced by AndrewS and Dom"
+echo
+echo " *** This script cannot uninstall any firmware automatically, only install"
+echo " *** different versions onto the system. Use with caution. For inexperienced"
+echo " *** users it is recommended to use their distro-provided packages (e.g."
+echo " *** 'raspberrypi-bootloader' or 'rasberrypi-bootloader-nokernel' along with a"
+echo " *** kernel image like 'linux-image-rpi' on raspbian) and NOT use rpi-update to"
+echo " *** avoid conflicts."
```

Comment:
For those inexperienced users, perhaps it would be better to reword part of the above as:
"For inexperienced users it is recommended to stick with their distro-provided packages and NOT use rpi-update, to avoid conflicts."